test/integration/test_client.c

```diff
@@ -145,3 +145,29 @@ TEST(client, querySql, setUp, tearDown, 0, client_params)
 
 	return MUNIT_OK;
 }
+
+/* Stress test of an EXEC_SQL with many ';'-separated statements. */
+TEST(client, semicolons, setUp, tearDown, 0, NULL)
```

Comment:
How do we know from this test that it is executing all of the statements separated by a semicolon instead of only the first one. Maybe we could change it to use an `INSERT` and count the total number of rows at the end.

Reply:
Seems like a good idea!

Reply:
Ah, a good trick here would be to add a single `INSERT` at the end and check that you can select it out again. That does the job of demonstrating that the whole string is executed.

Reply:
Done!

Reply:
You are right! Good idea, test is looking perfect now.